### PR TITLE
Add sidk_s5jt200 debug spec file

### PR DESCRIPTION
**Pull request by junmin-kim**

### CODE REVIEW

File: build/configs/sidk_s5jt200/.debugSpec.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    * PROJECT_PATH - current project path
+    * BOARD - selected board name
+    * SCRIPT_PATH - selected script path
+    * SCRIPT_FILE - selected script file
+-->
+<debugs>
+    <debug name="openocd debug" type="openocd">
+        <!-- path attribute value must be relative path from project root -->
+        <!-- usage : openocd -f debug_all.cfg -->
+        <options name="scriptfile" type="file" path="${PROJECT_PATH}/build/configs/${BOARD}/tools/openocd" filter="*.cfg"/>
```
Comment (sunghan-chang) [inline build/configs/sidk_s5jt200/.debugSpec.xml]: Is it *.cfg? not s5jt200_attach_gdb.cfg?

Comment (junmin-kim) [reply build/configs/sidk_s5jt200/.debugSpec.xml]: This code means all cfg files in ${PROJECT_PATH}/build/configs/${BOARD}/tools/openocd.
So user can select one of them.

Comment (sunghan-chang) [reply build/configs/sidk_s5jt200/.debugSpec.xml]: For debug, there is only one cfg file. Why do you think that we can use all of cfg for debug?

File: build/configs/artik053/.debugSpec.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
     <debug name="openocd debug" type="openocd">
         <!-- path attribute value must be relative path from project root -->
         <!-- usage : openocd -f debug_all.cfg -->
-        <options name="scriptfile" type="file" path="${PROJECT_PATH}/build/configs/${BOARD}/tools/openocd" filter="*.cfg"/>
+        <options name="scriptfile" type="file" path="${PROJECT_PATH}/build/configs/${BOARD}/tools/openocd" filter="artik053.cfg"/>
```
Comment (sunghan-chang) [inline build/configs/artik053/.debugSpec.xml]: This is not a "sidk_s5jt200 debug spec file"

**Pull request merged**